sources/lib/Resources/config/services/pomm.yml

```diff
@@ -8,10 +8,16 @@ services:
         arguments: ['@pomm']
         tags:
             - { name: request.param_converter, converter: pomm.entity }
+
     pomm.serializer.normalizer:
         class: 'PommProject\SymfonyBridge\Serializer\Normalizer\FlexibleEntityNormalizer'
         tags:
-            - { name: serializer.normalizer }
+            - { name: serializer.normalizer, priority: 999 }
```

Comment:
why `999`? I'm sure it can be a `<100` number that doesn't look "random" :stuck_out_tongue: no?